Packs/BmcHelixRemedyForce/Integrations/BmcHelixRemedyForce/BMCHelixRemedyforce.py

```diff
@@ -0,0 +1,2883 @@
+                readable_output=readable_output,
+                raw_response=api_response
+            ))
+
+    except Exception as e:
+        readable_output = HR_MESSAGES['CREATE_INCIDENT_WARNING'].format(create_result.get('Number', ''),
+                                                                        ", ".join(fields), str(e))
+        warning_output = create_result
+        context_output = {
+            OUTPUT_PREFIX['INCIDENT_WARNING']: warning_output
+        }
+        demisto.error(traceback.format_exc())  # print the traceback
+        return_warning(
+            message=readable_output,
+            exit=True,
+            warning=readable_output,
+            outputs=context_output,
+            ignore_auto_extract=True)
+
+
+@logger
+def bmc_remedy_incident_update_command(client, args):
+    args = remove_extra_space_from_args(args)
+    incident_number = args.pop('incident_number', '')
+    incident_number = incident_number[2:] if incident_number.startswith("IN") else incident_number
+
+    endpoint_to_get_id = SALESFORCE_QUERIES["GET_ID_FROM_SERVICE_REQUEST_NUMBER"].format(incident_number)
+
+    # Get id from incident number
+    response = client.http_request(method='GET', url_suffix=URL_SUFFIX["SALESFORCE_QUERY"],
+                                   params={'q': endpoint_to_get_id})
+    if response.get('records') and not response.get('records', [])[0].get('BMCServiceDesk__isServiceRequest__c'):
+        incident_id = response.get('records', [])[0].get('Id')
+    else:
+        raise DemistoException("{}".format(MESSAGES['NOT_FOUND_INCIDENT']).format(incident_number))
+
+    if not incident_id or incident_id.strip() == '':
+        raise ValueError(MESSAGES['NOT_FOUND_ERROR'])
+
+    # Prepare update request payload and get field names from additional arguments
+    update_payload, fields = get_update_incident_payload(args)
+
+    validate_incident_update_payload(update_payload)
+
+    context_output = {
+        "Id": incident_id,
+        "Number": incident_number
+    }
+    try:
+        id_suffix = '/{}'.format(incident_id)
+        update_api_response = client.http_request('PATCH', URL_SUFFIX['UPDATE_INCIDENT'] + id_suffix,
+                                                  json_data=update_payload)
+        context_output["LastUpdatedDate"] = datetime.now().strftime(DATE_FORMAT)
+
+        if isinstance(update_api_response, Response) and update_api_response.status_code == 204:
+            readable_output = HR_MESSAGES['UPDATE_INCIDENT_SUCCESS'].format(incident_number)
+
+            return_results(CommandResults(
+                outputs_prefix=OUTPUT_PREFIX['INCIDENT'],
+                outputs_key_field='Id',
+                outputs=context_output,
+                readable_output=readable_output,
+                raw_response=context_output
+            ))
+
+    except Exception as e:
+        readable_output = HR_MESSAGES['COMMAND_FAILURE'].format(demisto.command(), str(e))
```

Comment:
This should be handled in `main`, try/catch is block is not needed here

Reply:
Yes we would handle this.